ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+clang-format -style=file -i stl/inc/** stl/src/** 2>&1

```

Comment:
How do we know what version of clang-format is being used? Where does it come from?

Reply:
It comes from "that's how Billy set up the hardware this task runs on". I'm going to put the list of what I did on the wiki.

Reply:
https://github.com/microsoft/STL/wiki/Setting-up-an-Azure-Pipelines-Agent